goon/browserassets/css/emojib.css

```diff
@@ -0,0 +1,5 @@
+.em{height:1.5em;width:1.5em;background-position:center;background-repeat:no-repeat;background-size:contain;display:inline-block;vertical-align:middle;text-indent:100%;white-space: nowrap;overflow:hidden;}
```

Comment:
No reason to concat this like this, format it so its easier to read for the future